__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -281,29 +284,30 @@ const ControlledMultiSelect = (args: PropsFor<typeof MultiSelect>) => {
     >(null);
     return (
         <View style={{gap: spacing.xSmall_8}}>
```

Comment:
**nit:** It would be nice increasing the `gap` here to give more spacing between form fields.

https://5e1bf4b385e3fb0020b7073c-avxcxlgzkg.chromatic.com/?path=/story/packages-dropdown-multiselect--error-from-validation

<img width="354" alt="Screenshot 2024-12-19 at 10 50 21 AM" src="https://github.com/user-attachments/assets/da92972f-1975-4e52-b69e-78125f27678c" />

(Maybe this needs to be applied in the specific story)

Reply:
Good call, increased the spacing!